kivy_ios/toolchain.py

```diff
@@ -1457,6 +1467,9 @@ def update(self):
         parser = argparse.ArgumentParser(
                 description="Update an existing xcode project")
         parser.add_argument("filename", help="Path to your project or xcodeproj")
+        parser.add_argument("recipes", nargs="+", help="Recipes to update")
```

Comment:
Instead of using 2 arguments (`recipes` and `--add-custom-recipe`) is better to rely on custom recipe folder name.

Reply:
Two things:
What if the recipe folder has a different name such as `myrecipe_test2`?
The `build` command already requires recipe name + path so it stays consistent.

Reply:
The recipe name is defined by the folder name into which the recipe is contained.
The same logic also applies to recipes that are bundled into the toolchain, for both `kivy-ios` and `python-for-android`

Reply:
Sorry for the delay, I thought I had introduced a bug but it wast just my fault.